apps/odyssey-stats/src/page-middleware/layout.jsx

```diff
@@ -27,15 +26,13 @@ export const ProviderWrappedLayout = ( {
 			>
 				<QueryClientProvider client={ queryClient }>
 					<ReduxProvider store={ store }>
-						<MomentProvider>
```

Comment:
We don't need this anymore because we load the moment locale elsewhere, otherwise it'd break the formatting somewhere in the rendering